test/server/spec/bin/builder-core.spec.js

```diff
@@ -21,24 +22,14 @@ var run = require("../../../../bin/builder-core");
 
 var base = require("../base.spec");
 
-// Helpers
-// **Note**: It would be great to just stub stderr, stdout in beforeEach,
-// but then we don't get test output. So, we manually stub with this wrapper.
-var stdioWrap = function (fn) {
-  return function (done) {
-    base.sandbox.stub(process.stdout, "write");
-
-    var _done = function (err) {
-      process.stdout.write.restore();
-      done(err);
-    };
+var readOut = function () {
+  base.mockFs.restore();
+  return fs.readFileSync("stdout.log", { encoding: "utf8" });
```

Comment:
I can, it'll just make the assertions using it a lot more tedious. Is there an actual benefit? Mocha only runs one test at a time, but even if they were parallel, we never want multiple things trying to touch these files at the same time.